brigade.js

```diff
@@ -12,16 +12,43 @@ function build() {
         "cd /src",
         "yarn install",
         "yarn compile",
-        "yarn test",
+        "yarn test"
+    ];
+
+    return build;
+}
+
+function audit() {
```

Comment:
There is one moderate vulnerability that cannot be remediated at this time.

I figured as long as that's the case, we should separate the audit and the unit tests from one another. This way, it's more clear, at a glance, that tests are passing. fwiw, we did this a long time ago on Brigade itself.